From e8a447424bf0be272f591f7f368e9581528ec179 Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Date: Mon, 8 Feb 2016 11:29:55 +0100
Subject: [PATCH 0094/2241] arm64: update ARCH_MVEBU for Marvell Armada 7K/8K
 support

This commit updates the ARCH_MVEBU Kconfig option introduced for
Armada 3700 to also be used for the Armada 7K and 8K platforms, by:

 - Selecting the appropriate clock and irqchip drivers
 - Updating the help text to mention Armada 7K and 8K

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---
 arch/arm64/Kconfig.platforms | 11 ++++++++---
 1 file changed, 8 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/Kconfig.platforms b/arch/arm64/Kconfig.platforms
index eade8ab..a1e2220 100644
--- a/arch/arm64/Kconfig.platforms
+++ b/arch/arm64/Kconfig.platforms
@@ -2,9 +2,14 @@ menu "Platform selection"
 
 config ARCH_MVEBU
 	bool "Marvell EBU SoC Family"
-	help
-	  This enables support for Marvell EBU familly such as the
-	  Armada 3700 SoC Family.
+	select ARM_GIC_ODMI
+	select ARMADA_AP806_CORE_CLK
+	select ARMADA_AP806_RING_CLK
+	help
+	  This enables support for Marvell EBU familly, including:
+	   - Armada 3700 SoC Family
+	   - Armada 7K SoC Family
+	   - Armada 8K SoC Family
 
 config ARCH_BCM_IPROC
 	bool "Broadcom iProc SoC Family"
-- 
2.7.4

